### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mario-subo**
this beta release fixes the build for lib

### CODE REVIEW

**Review by kherP — APPROVED** [review]

File: lib/src/Textbox/Textbox.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { randomId } from "@sebgroup/frontend-tools";
+import { randomId } from "@sebgroup/frontend-tools/randomId";
```
Comment (omarVodiaK) [inline lib/src/Textbox/Textbox.tsx:2]: shouldn't it work like this
import { randomId } from '@sebgroup/frontend-tools';
```suggestion
import { randomId } from "@sebgroup/frontend-tools/randomId";
```

[the remark this answers is not in the document]
Comment (kherP) [reply lib/src/Textbox/Textbox.tsx:2]: that one actually including all the other functions, that's why we are specifying the randomId  path

**Pull request merged**